databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksConfig.java

```diff
@@ -114,6 +114,24 @@ public class DatabricksConfig {
   @ConfigAttribute(env = "DATABRICKS_RATE_LIMIT")
   private Integer rateLimit;
 
+  @ConfigAttribute(env = "PROXY_HOST")
+  private String proxyHost;
+
+  @ConfigAttribute(env = "PROXY_PORT")
+  private Integer proxyPort;
+
+  @ConfigAttribute(env = "PROXY_USERNAME")
+  private String proxyUsername;
+
+  @ConfigAttribute(env = "PROXY_PASSWORD")
+  private String proxyPassword;
+
+  @ConfigAttribute(env = "PROXY_AUTH_TYPE")
+  private ProxyConfig.ProxyAuthType proxyAuthType;
```

Comment:
How does this work if someone puts it in their `.databrickscfg`? You might need to add logic to `ConfigAttributeAccessor.setValueOnConfig`